### PR TITLE
Fixed bug where feedback text boxes caused horizontal scroll, moved some styles to emotion

### LINKED ISSUE #1057: Feedback text boxes cause horizontal scroll on small screens
screen shot 2018-08-29 at 1 07 37 pm

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1057

### CODE REVIEW

**Comment by sastels** [comment]
What part of the changes fixed the scroll bar issue? Looks like you just added more emotion.

File: components/feedbackBar.js [hunk 1]
```diff
@@ -45,13 +42,21 @@ const Inner = css`
     }
   }
 `;
-
 const TextHold = css`
   background-color: #505050;
   padding: 10px 0;
-  width: 400px;
```
Comment (smcmurtry) [inline components/feedbackBar.js:52]: @sastels this is what fixed it 😄

Comment (sastels) [reply components/feedbackBar.js:52]: that would do it 👍

**Review by sastels — APPROVED** [review]
🦑

**Pull request merged**